bootcamp-milestone-2/src/app/resume/page.tsx

```diff
@@ -0,0 +1,90 @@
+import style from "./page.module.css";
+
+
+export default function Resume() {
+    return (
+        <div className={style.joe}>
+            <main>
+                <h1 className="page-title">Resume</h1>
+                <br />
+                <a href="PlaceholderPage.pdf" download className="text-white"> Download Resume </a>
+
+                <div className="resume">
+                    <section className="section">
+                        <h2 className="section-title">
+                            Education
+                        </h2>
+                        <div className="entry">
```

Comment:
I would consider making a component for all these entries on your resume to reduce having to repeat code.